usecase/genome/mutator.go

```
@@ -5,7 +5,22 @@ import (
 	"math/rand"
 )
 
-func NewMutator(min, max, denom int) (*Mutator, error) {
+type Mutator interface {
+	mutate(b string) string
+}
+
+type NopMutator struct {
+}
+
+func NewNopMutator() (*NopMutator, error) {
```

this doesn't need to return an error